src/packages/emmett-sqlite/tsup.config.ts

```diff
@@ -0,0 +1,20 @@
+import { defineConfig } from 'tsup';
+
+const env = process.env.NODE_ENV;
+
+export default defineConfig({
+  splitting: true,
+  clean: true, // clean up the dist folder
+  dts: true, // generate dts files
+  format: ['cjs', 'esm'], // generate cjs and esm files
+  minify: true, //env === 'production',
```

Comment:
```suggestion
  minify: false, //env === 'production',
```

For some reason some bundlers have issues when we minify code. I disabled that recently.